common/src/main/resources/startup.properties

```diff
@@ -98,7 +98,7 @@
 #Configurations used in UTs
 debug.config.store.uri=file://${user.dir}/target/store
 #Location to store state of Feed SLA monitoring service
-debug.feed.sla.service.store.uri= file://${user.dir}/target/data/sla/pendingfeedinstances
+debug.entity.sla.service.store.uri= file://${user.dir}/target/data/sla/pendingfeedinstances
```

Comment:
The store location should be `file://${user.dir}/target/data/sla/pendingEntityInstances`